manifests/cloner/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM fedora:latest
```

Comment:
We should probably use a specific version so that it doesn't change underneath us.

Reply:
should it be fedora at all? Is fedora:28 good enough?

Reply:
Yes.  fedora:28 is good.

Reply:
just as a note: the cdi controller and importer currently use alpine but there's an issue to change this to fedora.